src/components/MultiLabelTextInput/MultiLabelTextInput.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+
+import { Chip, TextField } from "@material-ui/core";
+import { withStyles } from "@material-ui/styles";
+import { Autocomplete } from "@material-ui/lab";
+
+const StyledAutocomplete = withStyles({
+  root: {
```

Comment:
This should all be in a style file right?

Reply:
Same for StyledChip and Styled input's style stuff

Reply:
idk why I couldn't highlight lines for the comment but I specifically meant the styling content, not the with styles itself